### ISSUE TITLE
Runtime exception

### ISSUE THREAD

**Issue by abougouffa** [mickstar/Zoo-For-Zotero#123]
# Bug description
When I open the app but I'm not connected to internet,  two error messages shows up, **Error downloading Collections** and **Error downloading items**.

The app continue to work after that, but the error messages are quite annoying, I suggest to not display the message since the app can work with the locally stored database.

I'm using **Zoo for Zotero** on an e-Ink tablet (Onyx BOOX Air Note), and on my phone, both on **Android 10**, with local attachments directory. It works perfectly except for this small issue. I can reproduce the same behavior on the two devices.

Thank you for this beautiful app ❤️

# Version
2.9a

# Screenshots

![Screenshot_2021-01-16-00-57-33-496_com.mickstarify.zooforzotero.jpg](https://user-images.githubusercontent.com/3716399/104790012-9e734f00-5796-11eb-8990-b78dccffcb64.jpg)

![Screenshot_2021-01-16-00-57-29-173_com.mickstarify.zooforzotero.jpg](https://user-images.githubusercontent.com/3716399/104790028-ab903e00-5796-11eb-9f12-0fc8aef20aea.jpg)

**Comment by mickstar** [comment]
Good point, it should just show a "not connected to internet" message on
the UI.
